### ISSUE TITLE
No way to use current Firefox?

### ISSUE THREAD

**Issue by mnn** [kallisti-dev/hs-webdriver#157]
Using the hello world example I tried various versions of Selenium without any luck.

Selenium 2.53.1 (recommended in readme)
---
(snippet, the output was very long)
```
Caused by: org.openqa.selenium.WebDriverException: Failed to connect to binary FirefoxBinary(/usr/bin/firefox) on port 7055; process output follows:
```

app:
```
HttpExceptionRequest Request {
  host                 = "127.0.0.1"
  port                 = 4444
  secure               = False
  requestHeaders       = [("Accept","application/json;charset=UTF-8"),("Content-Type","application/json;charset=UTF-8")]
  path                 = "/wd/hub/session"
  queryString          = ""
  method               = "POST"
  proxy                = Nothing
  rawBody              = False
  redirectCount        = 10
  responseTimeout      = ResponseTimeoutDefault
  requestVersion       = HTTP/1.1
}
 ResponseTimeout
Received ExitFailure 1 when running
```

Selenium 3.141.59
---

app:
```
BadJSON "key \"ELEMENT\" not present"
Received ExitFailure 1 when running
```

Selenium 4.0.0-alpha-1
---

app:
```
BadJSON "key \"ELEMENT\" not present"
Received ExitFailure 1 when running
```

---

Firefox 67.0b2 (most recent in ubuntu repo)
Geckodriver: 0.24.0 (most recent version)
webdriver: 0.8.5 (most recent on stackage and hackage)

---

Am I missing something, or there is no working combination of Selenium and this library?

**Comment by matil019** [comment]
I believe #153 resolved that problem, and it hasn't been released yet.

**Comment by kallisti-dev** [comment]
[Now released on Hackage as part of webdriver-0.9](https://hackage.haskell.org/package/webdriver-0.9).
